pegasus/cache/i18n/en-US.yml

```diff
@@ -67,6 +67,7 @@
   congrats_next_tutorials_desc: "Try these shorter, 1 hour tutorials or try a <a href=\"%{current_path}/next\">partner tutorial</a>."
   get_a_certificate_of_achievement: "Get a certificate of achievement"
   view_course: "View course"
+  view_unit: "View unit"
```

Comment:
For these gsheet generated files, I'll make the change in the gsheet when ready to merge in.